src/app.js

```diff
@@ -53,10 +53,20 @@ app.use(session({
   store: await getRedisStore()
 }))
 
+// Load breaches into namespaced cache
+await (async () => {
+  try {
+    await loadBreachesIntoApp(app)
+  } catch (error) {
+    console.error('Error loading breaches into app.locals', error)
+  }
+})()
```

Comment:
I believe top-level `await` is available to us in this Node version, so you should be able to make this more readable?  e.g.
```
try {
  await loadBreachesIntoApp(app)
} catch (error) {
  console.error('Error loading breaches into app.locals', error)
}
```

Reply:
Thanks for this catch!